nomenclature/region_mapping_models.py

```diff
@@ -77,7 +82,14 @@ def create_from_region_mapping(cls, file: Union[Path, str]):
             schema = yaml.safe_load(f)
 
         # Validate the input data using jsonschema
-        validate(mapping_input, schema)
+        try:
+            validate(mapping_input, schema)
+        except ValidationError as e:
+            # Add file information in case of error
+            raise ValidationError(f"{e.message} in {file}")
+
+        # Add the file name to mapping_input
+        mapping_input["file"] = file
```

Comment:
Looking at this PR again, I guess it would be better to only store the relative path as attribute in the first place? (rather than doing that in the error messages)

Reply:
I was thinking about that too, if there's no value in keeping the full path (for debugging, error tracing, etc..) then I'll just change it to the relative path.

Reply:
I briefly tried saving only the relative path but that caused some tests to fail. I now included the conversion of the path in the `RegionNameCollisionError` class.